doc/tutorials/enable_s5.rst

```diff
@@ -22,48 +22,49 @@ S5 Architecture
 ***************
 
 ACRN provides a mechanism to trigger the S5 state transition throughout the system.
-It uses a vUART channel to communicate between the Service and User VMs.
+It uses a vUART channel to communicate between the Service VM and User VMs.
 The diagram below shows the overall architecture:
 
 .. figure:: images/s5_overall_architecture.png
    :align: center
    :name: s5-architecture
 
-   S5 overall architecture
+   S5 Overall Architecture
 
 - **Scenario I**:
 
     The User VM's serial port device (``ttySn``) is emulated in the
-    Device Model, the channel from the Service VM to the User VM:
+    Device Model. The channel from the Service VM to the User VM is as follows:
```

Comment:
The text on lines 36-40 is indented too far (two spaces).  Same problem on lines 44-40.